sympy/physics/vector/frame.py

```diff
@@ -554,45 +546,145 @@ def orient(self, parent, rot_type, amounts, rot_order=''):
         The following two lines show how the sense of the rotation can be
         defined. Both lines produce the same result.
 
-        >>> B.orient(N, 'Axis', (q1, -N.x))
-        >>> B.orient(N, 'Axis', (-q1, N.x))
+        >>> B.orient_axis(N, (q1, -N.x))
+        >>> B.orient_axis(N, (-q1, N.x))
+
+        """
+
+        from sympy.physics.vector.functions import dynamicsymbols
+        _check_frame(parent)
+
+        amounts = list(amounts)
+        for i, v in enumerate(amounts):
+            if not isinstance(v, Vector):
+                amounts[i] = sympify(v)
+        if not (isinstance(amounts, (list, tuple)) & (len(amounts) == 2)):
+            raise TypeError('Amounts are a list or tuple of length 2')
+        theta = amounts[0]
+        axis = amounts[1]
+        axis = _check_vector(axis)
+        parent_orient_axis = []
+
+        if not axis.dt(parent) == 0:
+            raise ValueError('Axis cannot be time-varying')
```

Comment:
So this check ensures the axis cannot be moving in the parent frame. So the language can be updated to be more clear in the docstring.